src/gluonts/dataset/repository/_uber.py

```diff
@@ -0,0 +1,82 @@
+import json
+import os
+from pathlib import Path
+from typing import Optional
+
+import pandas as pd
+
+from gluonts.dataset.repository._util import metadata
+
+
+def generate_uber_dataset(
+    dataset_path: Path, uber_freq: str, prediction_length: Optional[int] = None
+):
+    from gluonts.dataset.repository.datasets import default_dataset_path

```

Comment:
Couldn't we just have this import at the top of the file?

Reply:
That's because the `gluonts.dataset.repository.datasets` have to include `gluonts.dataset.repository._uber`.